recipes/r-ezplot/LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2020
+COPYRIGHT HOLDER: Wojtek Kostelecki
```

Comment:
No need for this file; the way it's specified in meta.yaml it'll grab the file coming with the tarball.

Reply:
Understood, thanks for the speedy reply!